src/main/java/hsleiden/stenentijdperk/stenentijdperk/Controllers/BoardController.java

```diff
@@ -163,8 +163,10 @@ public void endTurn() {
 
             if (!villagersLeft) {
                 boardmodel.setPhase(2);
+                FirebaseController.updateBoardFieldInt(String.valueOf(this.players.get(0).getLobby()), "phase", boardmodel.getPhase());
                 int turnCheck = (boardmodel.getTurn() - 1) % 4;
                 boardmodel.setPlayer(players.get(turnCheck));
+                FirebaseController.updateBoard(String.valueOf(this.getPlayer().getLobby()), this.boardmodel);
```

Comment:
Zoals ik het lees update je een field en daarna update je de hele board, dus is die eerste call onnodig.